synapse/tests/test_lib_node.py

```diff
@@ -256,3 +259,22 @@ class NodeTest(s_t_utils.SynTest):
                                                path=path))
                 self.eq(nodes[0][0].props.get('loc'), 'ca')
                 self.eq(path.vars.get('bar'), 'ru')
+
+    async def test_node_repr(self):
+
+        async with self.getTestCore() as core:
+
+            nodes = await core.nodes('[ inet:ipv4=1.2.3.4 :loc=us ]')
+            self.len(1, nodes)
+
+            node = nodes[0]
+
+            self.eq('1.2.3.4', nodes[0].repr())
+
+            self.eq('us', node.repr('loc'))
+
+            with self.raises(s_exc.NoSuchProp):
```

Comment:
we don't have to write these as context managers

Reply:
Any reason we wouldn't want to? They're cleaner and more readable than the other way :)

Reply:
since we're not using the context manager to examine the exception that occurs, we don't gain anything. we also are inconsistent with the voice of a bunch of other test code.